site/blog/deepseek-redteam.md

```diff
@@ -0,0 +1,183 @@
+---
+date: 2025-02-03
+image: /img/blog/deepseek-redteam/red_whale.png
+---
+
+# What are the Security Risks of Deploying DeepSeek-R1? 
+
+*Warning: This blog contains graphic content that may be disturbing to some readers.*
```

Comment:
You can disclaim and collapse only the harmful stuff below instead of keeping it all the way up here